designs/extensible-cli-and-scaffolding-plugins-phase-1-5.md

```diff
@@ -256,3 +311,11 @@ type CreateWebhookSubcommand interface {
 	Scaffolder
 }
 ```
+
+An additional interface defines the bundle method to return the wrapped plugins:
+```go
+type Bundle interface {
+	Plugin
+	Plugins() []Plugin
+}
+```
```

Comment:
```suggestion
```

**Note**: the bundle plugin is a mandatory solution to be addressed with 1.5 plugin implementation in order to address the SDK current requirements and behaviours. See that SDK CLI requires scaffold default plugins to do customizations on top for all its operator types plugins which are not achievable without this solution. 
 
```

I'd suggested we clarifies that as well.